src/state/profile/index.tsx

```diff
@@ -5,7 +5,7 @@ import { getProfile, getProfileAvatar, getUsername } from './helpers'
 
 const initialState: ProfileState = {
   isInitialized: false,
-  isLoading: true,
+  isLoading: false,
```

Comment:
It should be stay false until we call fetchProfile (which requires account) otherwise if there is no account connected It will stay loading true forever

Reply:
tested locally - seems to not cause any issues.
I wonder why it was true in the first place. 😑